app/helpers/pages_helper.rb

```diff
@@ -0,0 +1,2 @@
+module PagesHelper
+end
```

Comment:
What is this file for?

Reply:
What is a `helper`?

Reply:
A method to that shares reusable code in Rails view.

---
Question:
1. Can we overload or override the built-in helpers and custom helpers?

Reply:
This file is basically for you to put your custom helper. You can overwrite the built in helper. But then, you'll have a problem later when you expect the built in to do one thing but it does another cause of your overwrite.

So, don't. Ever. Ever. Overwrite built in stuffs.

This file is just a way for you to group helpers for pages views. It is by no means scoped to pages view alone. You can have another view but use the helpers defined here